botchat.css

```diff
@@ -279,6 +279,16 @@
         max-width: 50px;
     }
 
+    .wc-card.receipt span.title {
+        font-weight: bolder;
+        display: block;
```

Comment:
Can you use a div instead of a span, and remove display:block, for both of these